sdk/resourcemanager/azure-resourcemanager/pom.xml

```diff
@@ -187,7 +187,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-http-okhttp</artifactId>
-      <version>1.3.3</version> <!-- {x-version-update;com.azure:azure-core-http-okhttp;dependency} -->
+      <version>1.4.0</version> <!-- {x-version-update;com.azure:azure-core-http-okhttp;dependency} -->
```

Comment:
curious why these are getting updated in this PR. There should have been an increment PR that did this
/?